raiden_webui/__init__.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+RAIDEN_WEBUI_PATH = os.path.join(os.path.dirname(__file__), 'ui')
```

Comment:
May you use `os.path.abspath` on `__file__` just to be explicit/sure it'll be the absolute path for export? Mixing relative paths between packages may give some hard to debug problems here.